travis/install_grpc.sh

```diff
@@ -12,19 +12,19 @@ if [ "$grpc_dist" != "" ]; then
 fi
 
 # for python, we'll need the latest virtualenv and tox.
-# running gRPC requires the six package, any version will do.
+# running gRPC requires the six package, version >=1.10.
 if [ "$grpc_dist" != "" ]; then
-  pip install --upgrade --root $grpc_dist --ignore-installed virtualenv tox
-  pip install --root $grpc_dist --ignore-installed six
+  # Create a virtualenv, which also creates a virualenv-boxed pip.
+  virtualenv $grpc_dist/usr/local
+  $grpc_dist/usr/local/bin/pip install --upgrade --ignore-installed virtualenv tox six
```

Comment:
Did you do this to work-around the missing --root option for older pip binaries?